packages/vehicle-lifecycle-network/lib/vda.js

```diff
@@ -97,21 +97,19 @@ function scrapAllVehiclesByColour(scrapAllVehicles) {
     return getAssetRegistry(NS_D + '.Vehicle')
         .then(function (ar){
             assetRegistry = ar;
-            return queryNative(JSON.stringify(q));
+            return query('selectAllCarsByColour', {'colour':scrapAllVehicles.colour});
         })
         .then(function (resultArray) {
-            console.log('TP function received query result: ', JSON.stringify(resultArray));
             if (resultArray.length < 1 ) {
```

Comment:
should this throw if no matching colours are found? Seems quite dangerous if a user is pruning a list?

Reply:
Now I have removed this print. Actually this is an extra print - for debugging